sharpy/solvers/timeintegratorsjax.py

```diff
@@ -0,0 +1,209 @@
+import numpy as np
```

Comment:
New time integrator for the JAX solver, should have the same numerics as the existing implementation.